### PR TITLE
Small code enhancements

### PR DESCRIPTION
**Pull request by kirugan**

### CODE REVIEW

File: adapters/vm2core/vm2core.go [hunk 1]
```diff
@@ -43,29 +44,15 @@ func AdaptOrderedMessageToL1(message vm.OrderedL2toL1Message) *core.L2ToL1Messag
...
-	slices.SortFunc(messages, func(a vm.OrderedL2toL1Message, b vm.OrderedL2toL1Message) int {
-		switch {
-		case a.Order > b.Order:
-			return 1
-		case a.Order < b.Order:
-			return -1
-		default:
-			return 0
-		}
+	slices.SortFunc(messages, func(a, b vm.OrderedL2toL1Message) int {
```
Comment (pnowosie) [inline adapters/vm2core/vm2core.go:47]: I'm just curious 🤔 Would this not work?
```go
slices.SortFunc(messages, cmp.Compare)
```

File: utils/const.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package utils
+
+const (
+	MB = 1024 * 1024 // 1 Megabyte
```
Comment (pnowosie) [inline utils/const.go:4]: I'd prefer be explicit
```suggestion
	Megabyte = 1024 * 1024
```

**Review by pnowosie — APPROVED** [review]
Nice 👍

File: utils/const.go (hunk 2 above)
Comment (kirugan) [reply utils/const.go]: Initially I thought `MB` is better, but just realised that your suggestion is better to read. Renamed, thanks

**Review by IronGauntlets — APPROVED** [review]

**Pull request merged**